src/app/Models/User.php

```diff
@@ -169,13 +162,20 @@ public function delete()
 
         try {
             parent::delete();
-        } catch (\Exception $e) {
+        } catch (\Exception $exception) {
             throw new ConflictHttpException(__(
```

Comment:
add dedicated exception

Reply:
or remove the `delete` method and let the already imported `AvoidsDeletionConflicts` trait handle it?

Reply:
trait sounds good